sass/style.sass

```diff
@@ -1227,3 +1227,50 @@ textarea.deploy-key
 .content .variables-description p
 	max-width: none
 
+button.abort
```

Comment:
There's a lot of specific styling here, I reckon you should try to remove it and rely on `.btn` and only add what is absolutely necessary (float: right, etc?) here.